src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx

```diff
@@ -14,68 +14,3 @@ npx create-react-app amplify-datastore --use-npm
 cd amplify-datastore
 npx amplify-app@latest
 ```
-
```

Comment:
@chrisbonifacio - was the removal of this intentional?

Reply:
Was not intentional in this commit, this change was already merged in another PR: https://github.com/aws-amplify/docs/pull/4935

Would merging the same change have any affect? I can try to make a clean commit if need be